test/unit/definers/functions.test.ts

```diff
@@ -144,15 +144,14 @@ describe('defineMediaLibraryAssetFunction', () => {
       expect(fn.event.on).toEqual(['create', 'update', 'delete'])
     })
 
-    // coming soon!
-    // test('should allow for creating events with explicit include* toggles', () => {
-    //   const fn = defineMediaLibraryAssetFunction({
-    //     name: 'test',
-    //     src: 'test.js',
-    //     event: {on: ['update'], includeDrafts: true},
-    //   })
-    //   expect(fn.event.includeDrafts).toEqual(true)
-    // })
+    test('should allow for creating events with explicit include* toggles', () => {
+      const fn = defineMediaLibraryAssetFunction({
+        name: 'test',
+        src: 'test.js',
+        event: {on: ['update'], includeDrafts: true, resource},
+      })
+      expect(fn.event.includeDrafts).toEqual(true)
+    })
```

Comment:
Should we have another test to make sure that when it is not included that it's value is falsey?